astrodb_utils/tests/test_utils.py

```diff
@@ -30,45 +30,42 @@ def test_ingest_publications(db):
     )
 
 
+@pytest.mark.parametrize(
+    "source_data",
+    [
+        ({
+            "source": "Apple",
+            "ra": 10.0673755,
+            "dec": 17.352889,
+            "reference": "Refr20",
+        }),
+        ({
+            "source": "Orange",
+            "ra": 12.0673755,
+            "dec": -15.352889,
+            "reference": "Refr20",
+        }),
+        ({
+            "source": "Banana",
+            "ra": 119.0673755,
+            "dec": -28.352889,
+            "reference": "Refr20",
+        })]
+)
 @pytest.mark.filterwarnings(
     "ignore::UserWarning"
 )  # suppress astroquery SIMBAD warnings
-def test_ingest_sources(db):
+def test_ingest_sources(db, source_data):
     # TODO: Test adding an alt name
-    source_data1 = Table(
-        [
-            {
-                "source": "Apple",
-                "ra": 10.0673755,
-                "dec": 17.352889,
-                "reference": "Refr20",
-            },
-            {
-                "source": "Orange",
-                "ra": 12.0673755,
-                "dec": -15.352889,
-                "reference": "Refr20",
-            },
-            {
-                "source": "Banana",
-                "ra": 119.0673755,
-                "dec": -28.352889,
-                "reference": "Refr20",
-            },
-        ]
-    )
-
-    ingest_sources(
+    print(source_data)
+    ingest_source(
         db,
-        source_data1["source"],
-        ras=source_data1["ra"],
-        decs=source_data1["dec"],
-        references=source_data1["reference"],
-        raise_error=True,
+        source_data["source"],
+        ra=source_data["ra"],
+        dec=source_data["dec"],
+        reference=source_data["reference"],
+        use_simbad=True,
```

Comment:
Do we want `use_simbad=False` here since Apple, Orange, Banana don't exist there?

Reply:
True actually works now!  But I'll delete it since True is the default anyway.